python/test/gluon/test_frontend.py

```diff
@@ -165,8 +165,8 @@ def tensor_memory_kernel(layout: ttgl.constexpr, tmem_layout: ttgl.constexpr):
     slice2 = mem.slice(YBLOCK // 2, YBLOCK // 2)  # noqa: F841
 
     buffers = ttgl.nvidia.blackwell.allocate_tensor_memory(ttgl.float32, [2, XBLOCK, YBLOCK], tmem_layout)
-    for i in range(2):
-        buffers.index(i).load(layout)
+    for ivar in range(2):
```

Comment:
why did the name have the change here?

Reply:
In the IR it changed to `%i` and I just updated it to make the connection more easily noticeable.